hack/test-go.sh

```diff
@@ -45,8 +45,12 @@ OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${OS_ROOT}/hack/common.sh"
 source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/hack/lib/util/environment.sh"
+source "${OS_ROOT}/hack/lib/util/trap.sh"
+source "${OS_ROOT}/hack/lib/log/stacktrace.sh"
 cd "${OS_ROOT}"
-os::log::install_errexit
+
+os::util::trap::init_err
+os::log::stacktrace::install
```

Comment:
In other files, you are invoking `os::util::trap::init_err` and `os::log::stacktrace::install` *before* `cd "${OS_ROOT}"`.  I don't see that the discrepancy will matter with the current code, but the inconsistency kind of jumps out at me, and it could violate some subtle or future assumption.